tests/generic.rs

```diff
@@ -120,7 +120,68 @@ mod test_x509_parser_crl {
 		let (_, reason_code) = x509_revoked_cert.reason_code().unwrap();
 	 	assert_eq!(reason_code.0, revoked_cert.reason_code.unwrap() as u8);
 
+		// The issuing distribution point extension should be present and marked critical.
+		let issuing_dp_ext = x509_crl.extensions().iter()
+			.find(|ext| ext.oid == x509_parser::oid_registry::OID_X509_EXT_ISSUER_DISTRIBUTION_POINT)
+			.expect("failed to find issuing distribution point extension");
+		assert!(issuing_dp_ext.critical);
+		// TODO(XXX): x509-parser does not yet parse the CRL issuing DP extension for further examination.
```

Comment:
No need for `XXX` here, `TODO:` is enough

Reply:
Sounds good. Switched to `TODO:`.